client/blocks/recurring-payments-stats-nudge/actions.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Internal Dependencies
+ */
+import { getPreference } from 'state/preferences/selectors';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { savePreference } from 'state/preferences/actions';
+
+export const dismissNudge = () => ( dispatch, getState ) => {
+	const siteId = getSelectedSiteId( getState() );
+	const preference = getPreference( getState(), 'recurring-payments-dismissible-nudge' ) || {};
+
+	return dispatch(
+		savePreference( 'recurring-payments-dismissible-nudge', {
+			...preference,
+			...{
```

Comment:
Minor, but I think this object spread is unnecessary since `{...foo, ...{[color]: 'blue'}}` and `{...foo, [color]: 'blue'}` both produce the same result.

Reply:
I copied this code from the https://github.com/Automattic/wp-calypso/blob/master/client/blocks/upwork-banner/actions.js

Reply:
I think this is a pattern here :)

Reply:
🤔 Hm. I guess it's because using the spread makes it so that you could merge more properties if you had them. 

Anyway, this is fine if you like it. Just wanted to explain that it had no effect.